FindOnPage.md

```diff
@@ -0,0 +1,498 @@
+
+    // Registers an event handler for the ActiveMatchIndexChanged event.
+    // This event is raised when the index of the currently active match changes.
+    // This can happen when the user navigates to a different match or when the active match is changed programmatically.
+    // Parameter is the event handler to be added.
+    // Returns a token representing the added event handler. This token can be used to unregister the event handler.
+    HRESULT add_ActiveMatchIndexChanged(
+        [in] ICoreWebView2FindActiveMatchIndexChangedEventHandler* eventHandler,
+        [out] EventRegistrationToken* token);
+    // Unregisters an event handler from the ActiveMatchIndexChanged event.
+    // parameter is the token of the event handler to be removed, obtained during the registration of the event handler.
+    HRESULT remove_ActiveMatchIndexChanged([in] EventRegistrationToken token);
+}
+```
+
+
+### Setting Up Find Configuration with MIDL3
+
+To represent the given C# examples in a manner consistent with the behavior demonstrated 
+earlier in the chat and align them with an API design that could be described using 
+MIDL3 (noted as C# for formatting), let's formalize the design for a hypothetical 
+WebView2 Find operation API. This design will incorporate setting up a find configuration,
+ starting a find operation, handling find operation events, and navigating through find matches.
+
+### CoreWebView2 Find Configuration and Direction
+
+```csharp
+namespace Microsoft.Web.WebView2.Core
+{
+    public enum CoreWebView2FindDirection
+    {
+        Forward,
+        Backward
+    }
+
+    public class CoreWebView2FindConfiguration
+    {
+        public string FindTerm { get; set; }
+        public CoreWebView2FindDirection FindDirection { get; set; }
+        public bool IsCaseSensitive { get; set; }
+        public bool ShouldMatchWord { get; set; }
+    }
+}
+```
+
+### CoreWebView2 Find Interface
+
+```csharp
+namespace Microsoft.Web.WebView2.Core
+{
+
+    enum CoreWebView2FindDirection
+    {
+        Forward, 
+        Backward, 
+    };
+
+runtime CoreWebView2Find
+{
+    [interface_name("Microsoft.Web.WebView2.Core.ICoreWebView2Find")]
+    {
+        void StartFind(CoreWebView2FindConfiguration configuration);
+        void FindNext();
+        void FindPrevious();
+        void StopFind();
+        bool ShouldHighlightAllMatches { get; set; }
+        bool ShouldHighlightActiveMatch { get; set; }
+        bool UseCustomUI { get; set; }
+        int ActiveMatchIndex { get; }; 
+        int MatchCount { get; }; 
+    }
+}
+
+
+    runtimeclass CoreWebView2FindConfiguration {
+        [interface_name("Microsoft.Web.WebView2.Core.ICoreWebView2StagingFindConfiguration")]
+        {
+            String FindTerm { get; set; };
+            CoreWebView2FindDirection FindDirection { get; set; };
+            Boolean IsCaseSensitive { get; set; };
+            Boolean ShouldMatchWord { get; set; };
```

Comment:
This FindConfiguration is missing several properties that's in the COM IDL FindConfiguration.